app/controllers/wikis.js

```diff
@@ -0,0 +1,130 @@
+const axios = require('axios')
+const HttpStatus = require('http-status-codes')
+const WikiHelper = require('../utils/wikis-helper')
+const HANDLER = require('../utils/response-helper')
+const Organization = require('../models/Organisation')
+const { changeFileOnRemote, addPageToIndex, fetchPagesIndex, updatePagesIndex, getOpts, getOrgId } = WikiHelper
+
+const clientId = process.env.GITHUB_OAUTH_APP_CLIENTID
+const clientSecret = process.env.GITHUB_OAUTH_APP_CLIENTSECRET
+
+const githubAPI = 'https://api.github.com'
+let accessToken = null
+
+module.exports = {
+
+  getWikis: async (req, res, next) => {
+    try {
+      if (!accessToken) {
+        const Org = await Organization.find({}).lean().exec()
+        if (Org[0].wikis.accessToken) {
+          accessToken = Org[0].wikis.accessToken
+        }
+      }
+      if (!accessToken) {
+        res.status(HttpStatus.OK).json({ wikis: 'NO_ACCESS_TOKEN' })
+      } else {
+        res.status(HttpStatus.OK).json({ wikis: await addPageToIndex(await fetchPagesIndex(), 'Home') })
```

Comment:
Just a small change, can you please change "await addPageToIndex(await fetchPagesIndex(), 'Home')" to
const data = await addPageToIndex(await fetchPagesIndex(), 'Home'); then use data.
It would be more clearer for newbies